allspice/utils/bom_generation.py

```diff
@@ -0,0 +1,378 @@
+from __future__ import annotations
+import base64
+
+from dataclasses import dataclass
+import re
+import time
+from typing import Optional, Union
+
+from ..allspice import AllSpice
+from ..apiobject import Content, Ref, Repository
+from ..exceptions import NotYetGeneratedException
+
+
+@dataclass
+class SchdocComponent:
+    description: str
+    manufacturer: str
+    part_number: str
+    designator: str
+
+
+@dataclass
+class PcbComponent:
```

Comment:
Should this be `PcbdocComponent` to be consistent with `SchdocComponent`? Is this specific to the file type?

Reply:
No, it isn't specific to the file type. Thanks for catching it, I'll rename `SchdocComponent` to `SchematicComponent`.